### PR TITLE
New servlet and added functions

### PR DESCRIPTION
**Pull request by anerishah18**
It seems that I can't make GET/POST requests to the new servlet I've created until it is deployed (maybe because the script is a node.js file), so I've committed the servlet that I call from my script along with the FindTrendingCharities file that includes two more functions that I need to address the GET/POST requests. This is currently untested because I can't call the servlet yet.

### CODE REVIEW

File: Donation-platform/src/main/java/com/google/servlets/TagQueryServlet.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.model.Charity;
+import com.google.model.Tag;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+
+/** Servlet that handles requests for the trending page* */
+@WebServlet("/tag-query")
+public class TagQueryServlet extends HttpServlet {
```
Comment (charu0291) [inline Donation-platform/src/main/java/com/google/servlets/TagQueryServlet.java:33]: the name of this servlet is very confusing. documentation says trending page but name says tag. Should we rename it more meaningful? like TrendingPageScoreUpdateServlet

Comment (anerishah18) [reply Donation-platform/src/main/java/com/google/servlets/TagQueryServlet.java]: changed it

File: Donation-platform/src/main/java/com/google/FindTrendingCharities.java [hunk 2]
```diff
@@ -159,4 +159,15 @@ private double getTagTrendingScore(Collection<Tag> tags) throws Exception {
     }
     return (sumScores / numTags);
   }
+
+    public Collection<Tag> getTagsDb() throws Exception{
+        Collection<Tag> tags= db.getAllTags();
```
Comment (tonioshikanlu) [inline Donation-platform/src/main/java/com/google/FindTrendingCharities.java:164]: Nitpick: consistent spacing

Comment (anerishah18) [reply Donation-platform/src/main/java/com/google/FindTrendingCharities.java:164]: thanks for pointing it out. will adjust it.

**Comment by ryan-draper** [comment]
LGTM

**Pull request merged**